arctic-report-card/ARCRC.ttl

```diff
@@ -766,6 +936,152 @@ odo:ARCRC_00000068 rdf:type owl:Class ;
                    rdfs:seeAlso "https://glossary.ametsoc.org/wiki/Wind" .
 
 
+###  http://purl.dataone.org/odo/ARCRC_00000069
+odo:ARCRC_00000069 rdf:type owl:Class ;
+                   rdfs:subClassOf odo:ARCRC_00000040 ;
+                   obo:IAO_0000115 "The potential occurrence of a natural physical process that may adversely impact human or ecological systems (IPCC 2019)" ;
+                   obo:IAO_0000119 "glacier and permafrost related hazards as defined in the Arctic Report Card 2021 Essay at https://arctic.noaa.gov/report-card/report-card-2021/glacier-and-permafrost-hazards/" ;
+                   dc:creator <http://orcid.org/0009-0007-8384-5074> ;
+                   dc:date "2024-11-21T19:43:23Z"^^xsd:dateTime ;
+                   dc:description "This Key Variable is used to inform Frostbite, 'Glacier and Permafrost Hazards'" ;
+                   rdfs:label "glacier and permafrost hazards"@en . 
+
+
+###  http://purl.dataone.org/odo/ARCRC_00000070
+odo:ARCRC_00000070 rdf:type owl:Class ;
+                   rdfs:subClassOf odo:ARCRC_00000040 ;
+                   obo:IAO_0000115 "A carbon flux refers to the amount of carbon exchanged between carbon stocks over a specified time. In simple terms, it is the movement of carbon between land, oceans, atmosphere, and living things." ;
+                   obo:IAO_0000119 "carbon balance as defined at https://www.canr.msu.edu/news/carbon-fluxes-and-carbon-stocks#" ;
+                   dc:creator <http://orcid.org/0009-0007-8384-5074> ;
+                   dc:date "2024-11-21T19:43:23Z"^^xsd:dateTime ;
+                   dc:description "This Key Variable is used to inform Other Indicator, 'Arctic Carbon Cycling'" ;
+                   rdfs:label "carbon flux"@en ;
+                   skos:equivalentClass "http://purl.dataone.org/odo/ECSO_00001105" .
+
+
+###  http://purl.dataone.org/odo/ARCRC_00000071
+odo:ARCRC_00000071 rdf:type owl:Class ;
+                   rdfs:subClassOf odo:ARCRC_00000040 ;
+                   obo:IAO_0000115 "Data for each seal included collection date, species, sex, standard length, and blubber thickness." ;
+                   obo:IAO_0000119 "ice seal vital rates as defined in the Arctic Report Card 2024 Essay Ice Seals" ;
+                   dc:creator <http://orcid.org/0009-0007-8384-5074> ;
+                   dc:date "2024-11-21T19:43:23Z"^^xsd:dateTime ;
+                   dc:description "This Key Variable is used to inform Other Indicator, 'Ice Seals'" ;
+                   rdfs:label "ice seal vital rates"@en . 
```

Comment:
Why are the vital rates lumped together as one key variable, when multiple things really seem to be being measured? Should these be broken apart? I think this is worth a discussion with Sandy and team at some point to try to resolve just what a Key Variable really is.

Reply:
This author hadn't actually provided any key variables, so I came up with ice seal vital signs. I can ask them for clarification and maybe provide it in an update after the report card has been published next week.